apps/web/src/ee/billing/components/HighlightsRow.tsx

```diff
@@ -28,19 +28,17 @@ const highlights: PlanHighlights = {
   [ApiServiceLevelEnum.FREE]: [
     { text: 'Up to 30,000 events per month' },
     { text: '3 teammates' },
-    { text: '7 days activity feed' },
+    { text: '30 days Activity Feed' },
```

Comment:
Should include "retention" for consistency with above https://github.com/novuhq/novu/pull/6665/files#diff-5652bd36f7786eb2cb0cfba31f96dd81e31ed6727f4e09694cf0cd2fa269c120R128
```suggestion
    { text: '30 days Activity Feed retention' },
```